uint32 seq
float64 sensor_time
float64 ros_time
float64 corrected_timestamp
